qpc/insights/utils.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+#
+"""Utilities for the insights module."""
+
+from __future__ import print_function
+
+# pylint: disable=no-name-in-module,import-error
+from distutils.version import LooseVersion
+
+
+class InsightsCommands():
+    """Creates insights-client commands for qpc client."""
+
+    def __init__(self, dev=False):
+        """Set class variables.
+
+        Args:
+            dev (bool): used to build dev insights commands for local use.
+        """
+        self.dev = dev
+        # mime type for QPC report data for uploading to insights
+        self.content_type = 'application/vnd.redhat.qpc.deployments+tgz'
+
+    def build_base(self):
+        """Will create a base used for creating insights commands."""
+        if self.dev:
+            self.content_type = 'application/vnd.redhat.qpc.test+tgz'
```

Comment:
I added this incase we wanted a separate endpoint for development.

Reply:
We should use that to specify what we are sending.  In this case, we are sending the deployments report.